qt/aqt/utils.py

```diff
@@ -885,6 +886,33 @@ def openFolder(path: str) -> None:
             QDesktopServices.openUrl(QUrl(f"file://{path}"))
 
 
+def showinFolder(path: str) -> None:
+    if is_win:
+        call(["explorer", "/select,", f"file://{path}"])
+    elif is_mac:
+        script = f"""
+        tell application "Finder"
+            activate
+            select POSIX file '{path}'
+        end tell
+        """
+        call(osascript_to_args(script))
+    else:
+        # Just open the file in any other platform
+        with no_bundled_libs():
+            QDesktopServices.openUrl(QUrl(f"file://{path}"))
```

Comment:
Have you copied+pasted this code from somewhere?  From a quick read of the Qt docs, it looks like openUrl() should do what we want on all platforms?

Reply:
For local files, openUrl() opens the file in the default viewer if it can. But, we want it to highlight the file even when there is a viewer available.

Reply:
Sorry, I was under the mistaken impression that it would select the file, not open it directly.